cfme/tests/test_rest.py

```diff
@@ -10,6 +11,8 @@
 from utils.version import current_version
 from utils.virtual_machines import deploy_template
 from utils.wait import wait_for
+from cfme.services.catalogs.catalog_item import CatalogItem
+from cfme.automate.service_dialogs import ServiceDialog
```

Comment:
Maybe I am too pedantic but we tend to group the imports and sort them by alphabet, so you should put these import around line 7 and make them alphabetically sorted.